packages/geoprocessing/scripts/init/createProject.test.ts

```diff
@@ -132,4 +132,52 @@ describe("createProject", () => {
     expect(gpConfig.geoprocessingFunctions.length).toBeGreaterThan(0);
     expect(gpConfig.clients.length).toBeGreaterThan(0);
   }, 120000);
+
+  it("should create empty project with all defaults", async () => {
+    const projectName = "test-project-empty-defaults";
+    const projectPath = path.join(rootPath, projectName);
+    await createProject(
+      {
+        name: projectName,
+        description: "",
+        author: "",
+        email: "",
+        license: "UNLICENSED",
+        organization: "",
+        repositoryUrl: "",
+        region: "us-west-1",
+        templates: [],
+        planningAreaType: "other",
+        bboxMaxLat: -180,
+        bboxMinLat: 180,
+        bboxMaxLng: -90,
+        bboxMinLng: 90,
+        planningAreaId: "Test Area",
+        planningAreaName: "Samoa",
```

Comment:
I like the idea of an "all defaults" test.  Related, consider excluding the `planningAreaName`, and set `planningAreaNameQuestion` to be false.  Then it should by default, fall back back to assigning the `planningAreaId` as the name in basic.json.

Consider testing that basic.json contains the bbox value you expected for a full e2e.

Reply:
Good idea, I'll update the test